collection_requirements.rst

```diff
@@ -175,15 +175,13 @@ CI Testing
 ===========
 
 * You MUST run ``ansible-test sanity`` from the `latest stable ansible-base/ansible-core branch <https://github.com/ansible/ansible/branches/all?query=stable->`_. 
-* You MUST run CI against all versions of ``ansible-base``/``ansible-core`` that the collection supports.
```

Comment:
I vote that this stays

Reply:
As long as the example github action does this, I vote it stays too.

Reply:
It does

Reply:
Approved at today's meeting:  You MUST run CI against each of the "major versions" (2.10, 2.11, 2.12, etc)  of``ansible-base``/``ansible-core`` that the collection supports. (Usually the head of the stable-xxx branches.)

Reply:
@abadger ^ the approved version isn't part of this PR

Reply:
Ah wait, it ended up later in the document. Sorry for the noise!